devel/cl-quicklisp/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                cl-quicklisp
+categories          devel
+license             permissive
+github.setup        quicklisp quicklisp-bootstrap 2015-01-28 version-
+
+platforms           darwin
+maintainers         @jrjsmrtn openmaintainer
+description         Quicklisp is a library manager for Common Lisp
+long_description    ${description}. It works with your existing Common Lisp \
+                    implementation to download, install, and load any of \
+                    over 1,500 libraries with a few simple commands.
+
+homepage            https://www.quicklisp.org/beta/
+
+checksums           rmd160  b0d6482491853919566b4a6dc7e86c731f949cbe \
+                    sha256  be2c4bf2d3d630b82c6081d2c01563980464810552b343ef84f6bfe75003127e \
+                    size    13941
+
+use_configure       no
+build               {}
```

Comment:
So is this a script? See also https://trac.macports.org/wiki/UsingTheRightCompiler.

Reply:
Yes, it is. Quicklisp is meant to be installed at user-level and not at system-level, that's why the port doesn't do much and further instructions are given in the notes. It is similar to Debian's [`cl-quicklisp`](https://packages.debian.org/buster/all/cl-quicklisp/filelist) package.

I do not understand the reference to [UsingTheRightCompiler](https://trac.macports.org/wiki/UsingTheRightCompiler). Could you elaborate?

Reply:
Given that it is a script, the [UsingTheRightCompiler](https://trac.macports.org/wiki/UsingTheRightCompiler) reference does not apply.